exporters/metrics/build.gradle

```diff
@@ -18,15 +18,17 @@ description = 'Cloud Monitoring Exporter for OpenTelemetry'
 dependencies {
```

Comment:
It's okay for this PR, but in future please avoid mixing formatting changes with actual code changes.